flow/layers/fuchsia_layer_unittests.cc

```diff
@@ -440,7 +447,9 @@ TEST_F(FuchsiaLayerTest, DISABLED_PhysicalShapeLayersAndChildSceneLayers) {
   expected.push_back(scenic::NewSetLabelCmd(/*id=*/1, "fuchsia test root"));
   expected.push_back(scenic::NewSetTranslationCmd(/*id=*/1, {0, 0}));
   expected.push_back(scenic::NewAddChildCmd(/*id=*/1, /*child_id=*/2));
-  expected.push_back(scenic::NewSetOpacityCmd(/*id=*/2, kOneMinusEpsilon));
+
+  // TODO(dworsham)
```

Comment:
can you explain this and create a bug for it? why is removing this not a problem?

Reply:
Its OK here because the test is disabled anyways.  However new version of CL doesn't have this